src/components/Layout/Footer.tsx

```diff
@@ -286,50 +286,54 @@ export function Footer() {
         </div>
         <div className="flex flex-col">
           <FooterLink href="/learn" isHeader={true}>
-            Learn React
+            Impara React
           </FooterLink>
           <FooterLink href="/learn/">Quick Start</FooterLink>
-          <FooterLink href="/learn/installation">Installation</FooterLink>
+          <FooterLink href="/learn/installation">Installazione</FooterLink>
           <FooterLink href="/learn/describing-the-ui">
-            Describing the UI
+            Descrivere la UI
           </FooterLink>
           <FooterLink href="/learn/adding-interactivity">
-            Adding Interactivity
+            Aggiungere Interattività
+          </FooterLink>
+          <FooterLink href="/learn/managing-state">
+            Gestione dello State
+          </FooterLink>
+          <FooterLink href="/learn/escape-hatches">
+            Uscite di Emergenza
           </FooterLink>
-          <FooterLink href="/learn/managing-state">Managing State</FooterLink>
-          <FooterLink href="/learn/escape-hatches">Escape Hatches</FooterLink>
         </div>
         <div className="flex flex-col">
           <FooterLink href="/reference/react" isHeader={true}>
-            API Reference
+            API di Riferimento
```

Comment:
```suggestion
            API di riferimento
```
Eliminerei l'iniziale maiuscola da Riferimento, cosa ne pensi?